### ISSUE TITLE
Play Yeoman doesn't work with dependency injected router

### ISSUE THREAD

**Issue by akkie** [tuplejump/play-yeoman#79]
The new Play 2.4 build doesn't work with the dependency injected router. So if I define:

```scala
routesGenerator := InjectedRoutesGenerator
```

in my `build.sbt` then I get a compile error because the Yeoman controller is an object and not a class.

**Comment by anotherhale** [comment]
Will this be resolved soon?

**Comment by Shiti** [comment]
I will look into it this weekend.

**Comment by Shiti** [comment]
@akkie @anotherhale Can you share a sample application?

**Comment by anotherhale** [comment]
https://github.com/anotherhale/play-silhouette-angular-seed

**Comment by Shiti** [comment]
@anotherhale Although I was able to resolve the compile-time error for Yeoman, I was unable to test it.
Could you please make the following changes and check if the application works as expected?

Creating a controller class and calling that in the routes file.

```
class YeomanController extends Controller {
  def index = Yeoman.index
}
```

```
GET         /                                @controllers.YeomanController.index
```

**Comment by anotherhale** [comment]
@akkie I don't have time to look into this this week.  Can you take a look at this?

Thanks, Andy

**Comment by akkie** [comment]
Yes, I'll look into this tommorrow.

**Comment by Shiti** [comment]
Added support and released 0.8.1.
